pages/styles/portfolio.css

```diff
@@ -0,0 +1,163 @@
+#grid-container {
+    display: grid;
+    grid-template-columns: repeat(4, 300px);
+    grid-template-rows:  250px 250px;
+    justify-items: center;
+    grid-template-areas:
+    "header header header header"
+    "content content1 content2 content3"
+}
+
+#grid-container {
+    border: 5px solid #fde2f0;
+    border-radius: 5px;
+    background-color: white;
+}
+
+#item1 {
+    grid-column: span 3;
+    grid-area: header;
+} 
```

Comment:
This element this rule gets applied to is less an actual portfolio item, and more the "PROJECTS" title, so I think the name `#item1` is misleading. I'd probably go with `#projects-title` or something like this.